src/experiments/data/actions.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Internal dependencies
+ */
+import TYPES from './action-types';
+import { EXPERIMENT_NAME_PREFIX } from './constants';
+
+export function toggleExperiment( experimentName ) {
+	const storageItem = JSON.parse(
+		window.localStorage.getItem( EXPERIMENT_NAME_PREFIX + experimentName )
+	);
+
+	const newVariation =
+		storageItem.variationName === 'control' ? 'treatment' : 'control';
+
+	storageItem.variationName = newVariation;
+
+	window.localStorage.setItem(
+		EXPERIMENT_NAME_PREFIX + experimentName,
+		JSON.stringify( storageItem )
```

Comment:
@moon0326 I know this is a bit late, but it might also be worth updating the transient set by `Experimental_Abtest` when we are handling the experiment on the server side.
It sets this transient `abtest_variation_<experiment_name>` see here: https://github.com/woocommerce/woocommerce-admin/blob/main/includes/class-experimental-abtest.php#L145

Reply:
That's a good idea @louwie17 👍 

I'll make a new PR to support this.